instance/models/mixins/domain_names.py

```diff
@@ -168,7 +168,7 @@ def get_prefix_domain_names(self):
 
     def get_custom_domains(self):
         """
-        Returns a list of custom sub-domains configured for the instance.
+        Returns a list of custom (internal) sub-domains configured for the instance.
         """
         return [
             domain for domain in self.extra_custom_domains.split("\r\n") if domain.endswith(self.internal_lms_domain)
```

Comment:
Wouldn't ``.splitlines`` be a better option here.

Reply:
Yes.

Reply:
Just pushed some changes to hopefully make it clearer exactly when and why we're filtering extra_custom_domains.